fbw-a380x/src/systems/instruments/src/SDv2/Pages/Cruise/CruisePage.tsx

```diff
@@ -42,15 +44,15 @@ export class CruisePage extends DestroyableComponent<SdPageProps> {
   ];
 
   private readonly enginesFuelUsedDisplay = this.enginesFuelUsed.map((fu) =>
```

Comment:
FYI, `enginesFuelUsed` has leaked subscriptions (the consumer subjects). I can't leave comments on those lines for some reason.

Those subscriptions should be added to be paused / resumed (of applicable) and destroyed with the component.